db/migrate/20180612082738_move_download_users_to_users.rb

```diff
@@ -8,7 +8,7 @@ def change
         department: download_user.department,
         is_government: download_user.is_government,
         api_key: nil,
-        contactable: false
+        contactable: download_user.created_at.between?(Time.utc(2018, 0o5, 30, 18), Time.now)
```

Comment:
Is this a typo @gidsg ? Should be `Time.utc(2018, 05, 30, 18)`

Reply:
@dankmitchell linter made me do it m8 https://github.com/bbatsov/ruby-style-guide#numeric-literal-prefixes